yammbs/analysis.py

```diff
@@ -124,13 +126,82 @@ def get_rmsd(
     )
 
 
+def _shift_angle(angle: float) -> float:
+    """Shift angles by +/- 180 degrees if they're close to +/- 180."""
+    # handle being close to both 180 and -180
+    if abs(abs(angle) - 180) < 3:
+        # subtract 180 if ~180, subtract -180 if ~-180
+        return float(angle - numpy.sign(angle) * 180)
+    else:
+        # otherwise, return the angle as is
+        return float(angle)
+
+
+def _get_bond_length(
+    conformers: tuple[Chem.Conformer, Chem.Conformer],
+    atom1_index: int,
+    atom2_index: int,
+) -> tuple[float, ...]:
+    """Get the bond length between two atoms in a conformer."""
+    return tuple(
+        GetBondLength(
+            conformer,
+            atom1_index,
+            atom2_index,
+        )
+        for conformer in conformers
+    )
```

Comment:
This looks funny, but keep in mind I intend to switch it later to run on one conformer at a time (like #119), so I hope this ends up as

```suggestion
    return GetBondLength(
        conformer,
        atom1_index,
        atom2_index,
    )
```